mavros_msgs/msg/ESCTelemetry.msg

```diff
@@ -7,4 +7,4 @@
 
 std_msgs/Header header
 
-mavros_msgs/ESCTelemetryItem[] esc_telemetry
+mavros_msgs/ESCTelemetryItem[12] esc_telemetry
```

Comment:
That bad change as it will always send 12 values. I'd prefer `[<=12]` for ROS2 and keep as is for 1.